test/mail_ssl_letsencrypt.bats

```diff
@@ -318,7 +317,7 @@ function _should_be_equal_in_content() {
 function _get_service_logs() {
   local SERVICE=${1:-'mailserver'}
 
-  local CMD_LOGS=(docker exec "${TEST_NAME}" "supervisorctl tail ${SERVICE}")
+  local CMD_LOGS=(docker exec "${TEST_NAME}" "supervisorctl tail -3000 ${SERVICE}")
```

Comment:
I will try to remove this again and see what happens :) I think this may be because I introduced a bug you spotted above, and this addition masked it (?). Good point!

Reply:
Done with the latest push (19a661538442cd8abdccbfe67aff37ac50daef61 and the last 4 commits).